### PR TITLE
fixes 24 image doesn't resize

### PR DESCRIPTION
**Pull request by brandhawa99**
might want to look at the other option i put in the comments of  ```img.picture```

### CODE REVIEW

File: frontend/src/components/ImageSlideshow/Image/Image.module.css [hunk 1]
```diff
@@ -2,9 +2,20 @@
...
 
 img.picture {
-    max-width: 100%;
-    max-height: 40vh;
+    /**
+     *---------------------------------------------------
+     * change object-fit to conver to see another option
+     *----------------------------------------------------
+     */
+    object-fit:contain;
```
Comment (rdhillon1016) [inline frontend/src/components/ImageSlideshow/Image/Image.module.css]: contain is good (vertical pictures look really weird with object-fit:cover).

File: frontend/src/components/ImageSlideshow/Image/Image.module.css [hunk 2]
```diff
@@ -2,9 +2,20 @@
     flex: 1;
     display: flex;
     justify-content: center;
+    max-width: 426px;
+    max-height: 240px;
+    border: 1.5px var(--primary-accent) solid;
+    border-radius: 1.25rem;
```
Comment (rdhillon1016) [inline frontend/src/components/ImageSlideshow/Image/Image.module.css]: border is a bit off-putting

**Review by rdhillon1016 — CHANGES_REQUESTED** [review]

**Comment by brandhawa99** [comment]
i fucked some shit up trying to do the squash thing

**Comment by brandhawa99** [comment]
tried to do it from the command line

**Pull request closed without merging**